project-p3-forms-master/code/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Form</title>
+    <link rel="stylesheet" href="./style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+<!-- Build your site up as you want it! Make sure to use the form element and try to send that form to httpbin by adding action="https://httpbin.org/anything" and method="POST" to your form tag. -->
+ 
+<div class="menu">
+  <menu>
+    <a href="./index.html">Home</a>  
+    <a href="./2.arbeit.html">Arbeitserfahrung</a>
+    <a href="./3.ausbildung.html">Ausbildung</a>
+    <a href="./4.sprachen.html">Sprachen</a>
+    <a href="./5.weiteres.html">Weiteres</a>
+    <a href="./6.IT.html">IT Kentnisse</a>
+    <a href="./7.anhang.html">Anhang</a>
+  </menu>
+</div>
+
+<h1>ANHANG</h1>
+
+<form>
+  <action="https://httpbin.org/anything" method="POST"></action>
```

Comment:
`action` should be an attribute of the `form` element, like this:
`<form action="https://httpbin.org/anything" method="POST">`